src/twin/consent.rs

```diff
@@ -41,9 +41,17 @@ pub(crate) struct UserConsentCommand {
     pub user_consent: HashMap<String, String>,
 }
 
+type GeneralConsents = Vec<String>;
```

Comment:
only a comment: it's weird to have of a plural of "a general consent"

Reply:
will fix in one the next PR's